specification/iotcentral/resource-manager/Microsoft.IoTCentral/stable/2018-09-01/iotcentral.json

```diff
@@ -410,6 +410,86 @@
         "deprecated": false
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.IoTCentral/applicationTemplates/{applicationTemplateId}": {
+      "get": {
+        "tags": [
+          "Apps"
+        ],
+        "description": "Get a single application template.",
+        "operationId": "Apps_Template",
+        "x-ms-examples": {
+          "Apps_Template": {
+            "$ref": "./examples/Apps_Template.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/applicationTemplateId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The body contains the filtered IoT Central application template.",
+            "schema": {
+              "$ref": "#/definitions/AppTemplate"
+            }
+          },
+          "default": {
+            "description": "DefaultErrorResponse",
+            "schema": {
+              "$ref": "#/definitions/ErrorDetails"
+            }
+          }
+        },
+        "deprecated": false
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.IoTCentral/applicationTemplates": {
+      "get": {
+        "tags": [
+          "Apps"
+        ],
+        "description": "Get all available application templates.",
+        "operationId": "Apps_ListTemplates",
+        "x-ms-examples": {
+          "Apps_ListTemplates": {
+            "$ref": "./examples/Apps_Templates.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          },
+          {
+            "$ref": "#/parameters/subscriptionId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The body contains all the of the filtered IoT Central application templates.",
```

Comment:
>filtered [](start = 61, length = 8)

What does the word "filtered" refer to here? What filters the templates?

Reply:
it's for our internal references, I removed it